### DISCUSSION TITLE
bootstrap v5 vs google  Largest Contentful Paint

### DISCUSSION THREAD

**Discussion by falakkhursheed** [twbs/bootstrap#31709] [General] [answered]
I'm a website optimizer front end developer. I try a lot of tricks to solve the Largest Contentful Paint lighthouse but can't solve when I induced a bootstrap file in a project, the bootstrap file show in   Eliminate render-blocking resources Please help.

**Comment by patrickhlauke** [comment]
moving this to discussion section

**Comment by XhmikosR** [comment, marked as answer]
The render blocking warning is due to not loading CSS asynchronously. You can investigate adding critical CSS and loading the rest asynchronously.
